scripts/README.md

```diff
@@ -1,17 +1,86 @@
 # Scripts
 
-This folder contains scripts used for environment setups
+This folder contains scripts used for environment setup and configuration.
+
+```
+.
+├── initializeEs.js
+├── initializeExpanded.js
+├── migrateImages.js
+├── repairEs.js
+├── republish.js
+├── updateEs.js
+└── utils
+    ├── esUtils.js
+    └── republishUtils.js
+```
 
 # initializeEs.js
+This script will initialize your ElasticSearch environment. It should be run during your initial setup of the project.
+
+> From the project root directory run the following command. Note that you may need to change the ENV value used to match the environment declared in the `cms/pm2.config.json` file setup.
+
+```
+ENV=prd npm run initializeEs
+```
+
+Optionally, you can specify the following environment variables when running this script:
+* `ENV`: PM2 environment (i.e. `dev`, `prd`, etc.)
+* `CMS_CONFIG`: path to your CMS `pm2.config.js` file
+* `ES_INDEX`: models index name
+* `PROJECT_ID`: Arranger project id
+* `ES_HOST`: ElasticSearch host URL
+
+# initializeExpanded.js
+This script adds CMS support for Expanded models. It is a one-time script intended to initialize existing models with `expanded = true`, and does not need to be run on new setups as of [Release 1.6.2](https://github.com/nci-hcmi-catalog/portal/releases/tag/1.6.2).
+
+# migrateImages.js
+This script migrates existing modela images stored in Mongo to AWS. It is a one-time script intended to migrate existing model images to an S3 bucket, and does not need to be run on new setups as of [Release 1.5.1](https://github.com/nci-hcmi-catalog/portal/releases/tag/1.5.1).
 
-example:
+# repairEs.js
```

Comment:
What you wrote is correct, but there are other times this is required to run, specifically if the type of an existing property needs to be changed in the model index mapping. Since you can't change the type of fields in existing index, the only way to accomplish that sort of change is by deleting the index and rebuilding it, ie. this script.

Reply:
I believe this script will automatically run `republish` as its last step, so we should call out that all saved models will be republished when this is run.